.goreleaser.yml

```diff
@@ -34,3 +36,77 @@ changelog:
     exclude:
       - '^docs:'
       - '^test:'
+
+# .goreleaser.yaml
+brews:
+  -
+    # Name template of the recipe
+    # Default to project name
+    name: hazelcast-commandline-client
+
+    tap:
+      owner: hazelcast
+      name: homebrew-hz
+
+      # Optionally a token can be provided, if it differs from the token provided to GoReleaser
+      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
+
+    # Template for the url which is determined by the given Token (github, gitlab or gitea)
+    #
+    # Default depends on the client.
+    url_template: "https://github.com/hazelcast/hazelcast-commandline-client/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+
+    # The project name and current git tag are used in the format string.
+    commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
+
+    # Caveats for the user of your binary.
+    # Default is empty.
+    caveats: "How to use this binary"
+
+    # Your app's homepage.
+    # Default is empty.
+    homepage: "https://github.com/hazelcast/hazelcast-commandline-client"
+
+    # Template of your app's description.
+    # Default is empty.
+    description: "Hazelcast Command-line Client Project"
+
+    # SPDX identifier of your app's license.
+    # Default is empty.
+    license: "Apache-2.0"
+
+    # Setting this will prevent goreleaser to actually try to commit the updated
+    # formula - instead, the formula file will be stored on the dist folder only,
+    # leaving the responsibility of publishing it to the user.
+    # If set to auto, the release will not be uploaded to the homebrew tap
+    # in case there is an indicator for prerelease in the tag e.g. v1.0.0-rc1
+    # Default is false.
+    skip_upload: false
+
+    # Custom install script for brew.
+    # Default is 'bin.install "program"'.
+    install: |
+      bin.install "hzc"
+      bash_completion.install "extras/bash_completion.sh"
+      zsh_completion.install "extras/zsh_completion.zsh" => "_hzc"
+
+    # Custom post_install script for brew.
+    # Could be used to do any additional work after the "install" script
+    # Default is empty.
+    post_install: |
+      ohai "Example usages:
```

Comment:
What is ohai?

Reply:
I used it only because it is documented as a way to print a message for user https://docs.brew.sh/Formula-Cookbook#convenience-tools